rockcraft.yaml

```diff
@@ -51,4 +40,3 @@ parts:
             useradd -R $CRAFT_OVERLAY -M -r -g mongodb -u 1000 mongodb
         override-prime: |
             craftctl default
```

Comment:
Is it still necessary here (after removal chowm)?

Reply:
you are correct.